databuilder/publisher/elasticsearch_publisher.py

```diff
@@ -135,7 +135,7 @@ def _fetch_old_index(self):
             indices = self.elasticsearch_client.indices.get_alias(self.elasticsearch_alias).keys()
             return indices
         except NotFoundError:
-            LOGGER.warn("Received index not found error from Elasticsearch", exc_info=True)
+            LOGGER.warn("Received index not found error from Elasticsearch")
```

Comment:
Good change!

Maybe even let the warning sound less scary with e.g. adding “If this is first run on a clean Elasticsearch setup this is expected, so don’t be alarmed”

Reply:
yeap:) Reword the message a bit.